.golangci.yml

```diff
@@ -32,7 +32,7 @@ linters-settings:
   lll:
     line-length: 140
   goimports:
-    local-prefixes: github.com/golangci/golangci-lint
+    local-prefixes: github.com/golangci/golangci-lint github.com/golangci/golangci-lint/pkg/golinters/goanalysis
```

Comment:
Is this change necessary?

Reply:
https://github.com/golangci/golangci-lint/pull/842#discussion_r342517289 

Yup, it is as per above comment. However, I am not sure why it didn't report for existing files with the same import.

Reply:
I had a similar issue with golangci reporting not goimported but neither `gofmt -s` nor `goimports -w` made/reported any changes.

The solution was to let golangci itself fix the issue. The diff after removing the local prefix and running `golangci-lint run --fix` is:

```diff
diff --git a/.golangci.yml b/.golangci.yml
index e0e982c..2a1eec7 100644
--- a/.golangci.yml
+++ b/.golangci.yml
@@ -32,7 +32,7 @@ linters-settings:
   lll:
     line-length: 140
   goimports:
-    local-prefixes: github.com/golangci/golangci-lint github.com/golangci/golangci-lint/pkg/golinters/goanalysis
+    local-prefixes: github.com/golangci/golangci-lint
   gocritic:
     enabled-tags:
       - diagnostic
diff --git a/pkg/golinters/gomnd.go b/pkg/golinters/gomnd.go
index 9b52a9e..856c331 100644
--- a/pkg/golinters/gomnd.go
+++ b/pkg/golinters/gomnd.go
@@ -1,9 +1,10 @@
 package golinters
 
 import (
-	"github.com/golangci/golangci-lint/pkg/golinters/goanalysis"
 	"golang.org/x/tools/go/analysis"
 
+	"github.com/golangci/golangci-lint/pkg/golinters/goanalysis"
+
 	magic_numbers "github.com/tommy-muehle/go-mnd"
 )
 
```

I think this is a better soultion instead of adding a sub path as local prefix.

Reply:
@bombsimon wow you are right 💯, thanks for your review comment, I have updated for the same.